modules/network-observability-working-with-conversations.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-working-with-conversations_{context}"]
+= Working with conversation tracking
+As an administrator, you can you can group network flows that are part of the same conversation. A conversation is defined as a grouping of peers that are identified by their IP addresses, ports, and protocols, resulting in an unique *Conversation Id*. You can query conversation events in the web console. These events are represented in the web console as follows:
+
+** *Conversation start* happens when a connection is starting or TCP flag intercepted
+** *Conversation tick* occurs at every specified interval defined in flow collector `spec.processor.conversationHeartbeatInterval` while the connection is up.
+** *Conversation end* occurs when the `FlowCollector` `spec.processor.conversationEndTimeout` is reached or TCP flag intercepted.
+** *Flow* is the network traffic flow that occurs within the specified interval.
+
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` so that `spec.processor.logTypes`, `conversationEndTimeout`, and `conversationHeartbeatInterval` are set according to your observation needs. A sample configuration is as follows:
```

Comment:
Configure the `FlowCollector` CR so that the `spec.processor.logTypes`, `conversationEndTimeout`, and `conversationHeartbeatInterval` parameters...